src/ambassadors/classification.ts

```diff
@@ -0,0 +1,32 @@
+export enum Class {
+  Mammalia,
+  Aves,
+  Reptilia,
+  Amphibia,
+  Arachnida,
+  Diplopoda,
+  Insecta,
+  Malacostraca,
+}
+
+const classificationOrder = ["Mammals", "Birds", "Reptiles & Amphibians", "Invertebrates"] as const;
+
+export type Classification = (typeof classificationOrder)[number];
+
+export function getClassification(c: Class): Classification {
```

Comment:
```suggestion
export const getClassification(c: Class): Classification => {
```
nitpick, I think existing code here prefers using arrow functions